app/locales/en-us/translations.js

```diff
@@ -280,7 +277,7 @@ const translations = {
                         "label": "I react annoyed if another person steals the show from me."
                     }
                 },
-                "label": "15. Please rate the extent to which you agree or disagree with the following statements:",
+                "label": "16. Please rate the extent to which you agree or disagree with the following statements:",
```

Comment:
Wondering if it makes more sense to just use the [`{{index}} variable`](http://stackoverflow.com/questions/19795562/accessing-index-in-each-in-emberjs) available in the `#each` context? That way if these are every reordered this doesn't have to be rewritten.